recipe/meta.yaml

```diff
@@ -25,12 +25,12 @@ requirements:
     - {{ compiler('c') }}                 # [not win]
     - {{ compiler('m2w64_c') }}           # [win]
     - {{ compiler('rust') }} >=1.56.1
-    - maturin
+    - maturin <0.15  # TODO: remove restriction after next sourmash release
```

Comment:
The changes for `maturin >= 0.15` are merged in sourmash for the next release, but `4.8.2` is broken